_posts/2019-01-02-35c3ctf-2018-krautflare.md

```diff
@@ -0,0 +1,846 @@
+---
+title: 'Krautflare'
+author:
+  - Jay Bosamiya (f0xtr0t)
+layout: post
+categories:
+  - Pwn
+ctf: 35c3
+year: 2018
+---
+
+This is a write-up for **krautflare** challenge, a browser pwnable challenge from [35c3ctf](https://ctftime.org/event/718). _You can also read from the [original post](https://www.jaybosamiya.com/blog/2019/01/02/krautflare/)_.
+
+{% comment %} The 100% hardcoded here feels wrong. We should probably
+have CSS that auto manages images and allows us to use the ! [] ()
```

Comment:
We shouldn't need to have 100% hardcoded images, but unfortunately we can't use the `! [] ()` syntax because we do not know what the base path of the file is (for instance it could be `/` or `/pwning.github.io/`). However, what we can do is use somewhat annoying jekyll syntax to do `{{ '/blahblahblah' | prepend: site.baseurl }}`.

Reply:
Ah, I wasn't really referring to that. I was referring to the width that is hardcoded as 100%. I know that in general, we may not know the base path, which is annoying.